modules/tinymce/src/core/test/ts/browser/html/StylesTest.ts

```diff
@@ -144,6 +144,84 @@ describe('browser.tinymce.core.html.StylesTest', () => {
     );
   });
 
+  it('TINY-9843: Should not compress styles', () => {
+    const styles = Styles();
+
+    assert.equal(
```

Comment:
Creating a function to abstract this `assert.equal(styles.serialize(styles.parse()))` code pattern would probably be good.

Reply:
Done